RoddyCore/src/de/dkfz/roddy/config/loader/ConfigurationFactory.groovy

```diff
@@ -875,7 +878,7 @@ class ConfigurationFactory {
         //any directory to "path". In case of the output directories, this was a bad thing! So we know about
         //this specific type of variable and just set it to path at any time.
         if (key.endsWith("OutputDirectory"))
-            type = "path"
+            type = CVALUE_TYPE_PATH
```

Comment:
I was wondering. What is really meant by "path" is "directory". Maybe we should user `CVALUE_TYPE_DIRECTORY` and introduce a "directory" identifier into the XMLs and then in version 4 use "directory" in the XMLs?

Reply:
maybe "dirname", "directoryname" (in analogy to "filename"; I'd prefer camelcase though), "directoryName"

Reply:
I'll make a Roddy 4 issue out of this.